src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWP.cs

```diff
@@ -542,6 +542,10 @@ internal string LogoPathFromUri(string uri)
                     var prefix = path.Substring(0, end);
                     var paths = new List<string> { path };
 
+                    // TODO: This value must be set in accordance to the WPF theme (work in progress).
+                    // Must be set to `contrast-white` for light theme and to `contrast-black` for dark theme to get an icon of the contrasting color.
+                    var theme = "contrast-black";
```

Comment:
Is this something you are planning to do in this PR? The work for theme is already in master.

Reply:
As discussed, we shall do it as part of another PR as the icons are loaded during indexing UWP apps so we might have to load both the light and dark themed icons and then have some type of a callback to inform the plugins about the theme so that they can decide accordingly. Issue #4122 is tracking this.